src/components/ResourceOverview/HospitalOverviewPage.tsx

```diff
@@ -0,0 +1,293 @@
+
+  const handleClickDelete = () => {
+    const hospitalId = selectedHospital;
+    deleteHospital({ variables: { id: hospitalId } });
+    setOpenModal(false);
+  };
+
+  const cells = hospitals.map((hospital: Hospital) => {
+    return (
+      <TableRow key={hospital.id}>
+        <TableCell classes={{ root: dRow.root }}>{hospital.name}</TableCell>
+        <TableCell classes={{ root: optionBtn.root }}>
+          <IconButton data-id={hospital.id} onClick={handleClickOptions}>
+            <MoreHorizIcon style={{ color: Colours.Black }} />
+          </IconButton>
+        </TableCell>
+      </TableRow>
+    );
+  });
+
+  return (
+    <div className={classes.wrapper}>
+      <Typography variant="h5">Hospital Overview</Typography>
+      <Typography variant="body2" classes={{ body2: paraStyle.body2 }}>
+        A list of all hospitals that can be added to an event.
+      </Typography>
+
+      <TableContainer>
+        <Table classes={{ root: table.root }}>
+          <TableHead>
+            <TableRow>
+              <TableCell classes={{ root: hRow.root }}>Hospital Name</TableCell>
+              <TableCell />
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {cells}
+            <Popper
+              id={id}
+              open={open}
+              popperOptions={{
+                modifiers: { offset: { enabled: true, offset: '-69.5,0' } },
+              }}
+              anchorEl={anchorEl}
+            >
+              <div>
+                <Table className={classes.tablePopper}>
+                  <TableBody>
+                    <TableRow
+                      hover
+                      classes={{ hover: optionBtn.menuHover }}
+                      onClick={handleClickEdit}
+                    >
+                      <TableCell classes={{ root: optionBtn.menuCell }}>
+                        <Typography variant="body2">Edit</Typography>
+                      </TableCell>
+                    </TableRow>
+                    <TableRow hover onClick={() => setOpenModal(true)}>
+                      <TableCell classes={{ root: optionBtn.menuDelete }}>
+                        <Typography variant="body2">Delete</Typography>
+                      </TableCell>
+                    </TableRow>
+                  </TableBody>
+                </Table>
+              </div>
+            </Popper>
+          </TableBody>
+        </Table>
+      </TableContainer>
+      <Dialog classes={{ paper: dialogStyle.paper }} open={openModal}>
+        <DialogTitle classes={{ root: dialogStyle.dialogTitle }}>
+          <Typography variant="h6">
+            <strong>You are about to delete a team member.</strong>
+          </Typography>
+        </DialogTitle>
+        <DialogContent classes={{ root: dialogStyle.dialogContent }}>
+          <Typography variant="body2">
+            Deleted team members will no longer have access to any casualty
+            collection points.
+          </Typography>
```

Comment:
This copy needs to be updated for hospitals (shouldn't say team member).

![image](https://user-images.githubusercontent.com/22457859/89363706-d4b48a80-d69e-11ea-9315-593d72048833.png)

Reply:
Also I would recommend avoiding the `strong` tag as you don't have complete control of what that does. Also please double check that it's necessary, as using the strong tag changes the text so that it doesn't align with the component library in figma.